backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5684510_sys_gh15086_JobRouter_Instance_ID.sql

```diff
@@ -0,0 +1,115 @@
+-- Field: Invoice (Vendor)_OLD(183,D) -> Invoice(290,D) -> Externe ID
+-- Column: C_Invoice.ExternalId
+-- 2023-04-06T20:50:31.861Z
+INSERT INTO AD_Field (AD_Client_ID,AD_Column_ID,AD_Field_ID,AD_Org_ID,AD_Tab_ID,ColumnDisplayLength,Created,CreatedBy,DisplayLength,EntityType,IncludedTabHeight,IsActive,IsDisplayed,IsDisplayedGrid,IsEncrypted,IsFieldOnly,IsHeading,IsReadOnly,IsSameLine,Name,SeqNo,SeqNoGrid,SortNo,SpanX,SpanY,Updated,UpdatedBy) VALUES (0,572375,713823,0,290,0,TO_TIMESTAMP('2023-04-06 21:50:30','YYYY-MM-DD HH24:MI:SS'),100,0,'D',0,'Y','Y','Y','N','N','N','N','N','Externe ID',0,340,0,1,1,TO_TIMESTAMP('2023-04-06 21:50:30','YYYY-MM-DD HH24:MI:SS'),100)
+;
+
+-- 2023-04-06T20:50:31.966Z
+INSERT INTO AD_Field_Trl (AD_Language,AD_Field_ID, Description,Help,Name, IsTranslated,AD_Client_ID,AD_Org_ID,Created,Createdby,Updated,UpdatedBy,IsActive) SELECT l.AD_Language, t.AD_Field_ID, t.Description,t.Help,t.Name, 'N',t.AD_Client_ID,t.AD_Org_ID,t.Created,t.Createdby,t.Updated,t.UpdatedBy,'Y' FROM AD_Language l, AD_Field t WHERE l.IsActive='Y'AND (l.IsSystemLanguage='Y' OR l.IsBaseLanguage='Y') AND t.AD_Field_ID=713823 AND NOT EXISTS (SELECT 1 FROM AD_Field_Trl tt WHERE tt.AD_Language=l.AD_Language AND tt.AD_Field_ID=t.AD_Field_ID)
+;
+
+-- 2023-04-06T20:50:32.087Z
+/* DDL */  select update_FieldTranslation_From_AD_Name_Element(543939) 
+;
+
+-- 2023-04-06T20:50:32.213Z
+DELETE FROM AD_Element_Link WHERE AD_Field_ID=713823
+;
+
+-- 2023-04-06T20:50:32.329Z
+/* DDL */ select AD_Element_Link_Create_Missing_Field(713823)
+;
+
+-- 2023-04-06T20:51:11.385Z
+/* DDL */  select update_FieldTranslation_From_AD_Name_Element(582231) 
+;
+
+-- 2023-04-06T20:51:11.494Z
+DELETE FROM AD_Element_Link WHERE AD_Field_ID=713823
+;
+
+-- 2023-04-06T20:51:11.586Z
+/* DDL */ select AD_Element_Link_Create_Missing_Field(713823)
+;
+
+-- UI Element: Invoice (Vendor)_OLD(183,D) -> Invoice(290,D) -> main -> 20 -> Externe ID
+-- Column: C_Invoice.ExternalId
+-- 2023-04-06T20:53:16.849Z
+INSERT INTO AD_UI_Element (AD_Client_ID,AD_Field_ID,AD_Org_ID,AD_Tab_ID,AD_UI_ElementGroup_ID,AD_UI_Element_ID,AD_UI_ElementType,Created,CreatedBy,IsActive,IsAdvancedField,IsAllowFiltering,IsDisplayed,IsDisplayedGrid,IsDisplayed_SideList,IsMultiLine,MultiLine_LinesCount,Name,SeqNo,SeqNoGrid,SeqNo_SideList,Updated,UpdatedBy,WidgetSize) VALUES (0,713823,0,290,540226,616643,'F',TO_TIMESTAMP('2023-04-06 21:53:15','YYYY-MM-DD HH24:MI:SS'),100,'Y','N','N','Y','N','N','N',0,'Externe ID',50,0,0,TO_TIMESTAMP('2023-04-06 21:53:15','YYYY-MM-DD HH24:MI:SS'),100,'S')
```

Comment:
Also consider moving to advanced view. Most of our users are not interested in this field, hence we do not have it displayed at all.